src/applications/disability-benefits/526EZ/components/ButtonContainer.jsx

```diff
@@ -59,6 +59,7 @@ export default function ButtonContainer(props) {
         <a
           className="usa-button-primary"
           href="https://www.ebenefits.va.gov/ebenefits/about/feature?feature=disability-compensation"
+          target="_blank"
```

Comment:
We should also add `rel="noopener"`. It's a minor risk given that this is gov site, but we don't want other sites to have access to the `window` on our pages.